apertium-eng.eng.rlx

```diff
@@ -328,7 +333,13 @@ SELECT (adv itg) IF (-1 BOS) ;
 SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
-
+SELECT Adv IF (0 Adj) (-1 Vblex) ;
```

Comment:
When is "on" an adjective?

Reply:
On's an adjective in cases like:
`Is the show still on?`

I think this rule should work.

Reply:
I think this might be a error, though, because "on" as an adjective isn't in the English bidix.  I think I meant "pr" in this case...